specification/monitor/resource-manager/readme.go.md

```diff
@@ -130,4 +136,58 @@ Please also specify `--go-sdk-folder=<path to the root directory of your azure-s
 
 ``` yaml $(tag) == 'package-2021-09' && $(go)
 output-folder: $(go-sdk-folder)/services/preview/monitor/mgmt/2021-09-01-preview/$(namespace)
-```
\ No newline at end of file
+```
+
+### Tag: package-2022-03-preview-datacollection and go
+
+These settings apply only when `--tag=package-2022-03-preview-datacollection --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2022-03-preview-datacollection' && $(go)
+output-folder: $(go-sdk-folder)/services/preview/monitor/mgmt/2022-03-26-preview/$(namespace)
```

Comment:
here the variable namespace is just `monitor`, we need to change it to `datacollection`
```suggestion
output-folder: $(go-sdk-folder)/services/preview/monitor/mgmt/2021-09-01-preview/datacollection
```

Reply:
I'll use the tag package-2022-02-01-preview-only and remove package-2022-03-preview-datacollection since both have the same spect list